exoscale/loadbalancer_test.go

```diff
@@ -561,18 +550,18 @@ func (ts *exoscaleCCMTestSuite) Test_loadBalancer_fetchLoadBalancer() {
 
 	// Non-existent NLB
 
-	ts.p.client.(*exoscaleClientMock).
-		On("GetNetworkLoadBalancer", ts.p.ctx, ts.p.zone, "lolnope").
-		Return(new(egoscale.NetworkLoadBalancer), errLoadBalancerNotFound)
-
-	_, err = ts.p.loadBalancer.(*loadBalancer).fetchLoadBalancer(ts.p.ctx, &v1.Service{
-		ObjectMeta: metav1.ObjectMeta{
-			Annotations: map[string]string{
-				annotationLoadBalancerID: "lolnope",
-			},
-		},
-	})
-	ts.Require().ErrorIs(err, errLoadBalancerNotFound)
+	// ts.p.client.(*exoscaleClientMock).
+	// 	On("GetLoadBalancer", ts.p.ctx, "lolnope").
+	// 	Return(new(v3.LoadBalancer), errLoadBalancerNotFound)
+
+	// _, err = ts.p.loadBalancer.(*loadBalancer).fetchLoadBalancer(ts.p.ctx, &v1.Service{
+	// 	ObjectMeta: metav1.ObjectMeta{
+	// 		Annotations: map[string]string{
+	// 			annotationLoadBalancerID: "lolnope",
+	// 		},
+	// 	},
+	// })
+	// ts.Require().ErrorIs(err, errLoadBalancerNotFound)
```

Comment:
We don't need this test?

Reply:
FIXED: 
https://github.com/exoscale/exoscale-cloud-controller-manager/pull/97/commits/aa47c6e4c96139054337887126045b6f20741c0f